smdebug/tensorflow/base_hook.py

```diff
@@ -437,7 +437,7 @@ def _write_for_tensor(self, tensor_name, tensor_value, save_collections, tensor_
 
         # this tensor_name is tf tensor name, need to convert to export_name
         tensor_ref = self._get_tensor_ref(tensor_name, save_collections=save_collections)
-        if tensor_ref:
+        if tensor_ref is not None:
```

Comment:
Unrelated to PR.
This if check sometimes leads to an annoying error during development.
See: https://stackoverflow.com/questions/7816363/if-a-vs-if-a-is-not-none for more details